File: translations/codenet/deepseek_commented/C++2P/deepseek/Code/s589139355.py
def main():
    a = int(input())
    t = 0
    h = 0

    for _ in range(a):
        taro, hana = input().split()
        
        if taro > hana:
            t += 3
        elif hana > taro:
            h += 3
        else:
            t += 1
            h += 1

    print(t, h)

main()

File: translations/codenet/uncommented/C++2J/starcoder/Code/s589139355.java

import java.util.*;
public class Main {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int a,t=0,h=0,lt,lh,tt,hh,l;
        a = in.nextInt();
        for(int i=0; i<a; ++i){
            String taro = in.next();
            String hana = in.next();
            lt= taro.length();
            lh= hana.length();
            if(lt>lh){
                l=lt;
            }
            else{
                l=lh;
            }
            tt=0;
            hh=0;

            if(taro.compareTo(hana)>0){
                t=t+3;
            }
            else if(hana.compareTo(taro)>0){
                h=h+3;
            }
            else if(hana.compareTo(taro)==0){
                h=h+1;
                t=t+1;
            }
        }
        System.out.println(t+" "+h);
    }
}
